charts/sysdig/values.yaml

```diff
@@ -255,7 +255,7 @@ nodeAnalyzer:
   # * SaaS US Web: us2.app.sysdig.com
   # * SaaS European Union: eu1.app.sysdig.com
   # * On-Prem: sysdig.my.company.com
-  # collectorEndpoint: sysdig.my.company.com
+  # apiEndpoint: sysdig.my.company.com
```

Comment:
Renamed to apiEndpoint. The onboarding command must be updated once this chat is released, although the old `collectorEndpoint` value will still work.